doc/cookbook/source/examples/classifier/kernel_svm.rst

```diff
@@ -0,0 +1,51 @@
+=============================
+Kernel Support Vector Machine
+=============================
+
+Kernel Support Vector Machine is a binary classifier which finds a hyper-plane in a space induced by a kernel, such that the margins between the two classes are maximized. The loss function that needs to be minimized is:
```

Comment:
"a data-separating hyperplane in a Hilbert space induced by a positive definite kernel. The hyperplane is chosen to maximize the margins between the two classes."